src/db/cdir_events_dilemma_choice_details_tables_vco.tsv

```diff
@@ -1,17 +1,19 @@
 dilemma_key	choice_key	associated_audio_trigger
 #cdir_events_dilemma_choice_details_tables;3;db/cdir_events_dilemma_choice_details_tables/vco		
-vco_brt_alb_dilemma_mananns_favour	FIRST
-vco_brt_alb_dilemma_mananns_favour	SECOND
+vco_brt_alb_dilemma_mananns_favour	FIRST	
```

Comment:
There is a lot of bullshit like this that gets in the middle of the review. Ideally we could merge one first to check the other ones properly, but overall it looks good to me.